templates/etc/haproxy/haproxy.cfg.j2

```diff
@@ -6,6 +6,8 @@ global
 defaults
 {% include 'defaults.cfg.j2' %}
 
+{% include 'peers.cfg.j2' %}
```

Comment:
all the examples I've seen have it below 'defaults' but happy to move it if required?